dyn_em/module_advect_em.F

```diff
@@ -10652,10 +10664,6 @@ PROGRAM feeder
     fzp(k) = .5* dnw(k  )/dn(k)
     fzm(k) = .5* dnw(k-1)/dn(k)
    ENDDO
-   DO k = kts,kte
```

Comment:
revert this change

Reply:
This statement is not in my code, nor in the code that I submitted for pull.....???????